### PR TITLE
Polish Chapter1

### PR DESCRIPTION
**Pull request by Crazycolorz5**

### CODE REVIEW

**Comment by Crazycolorz5** [comment]
Not yet ready for review.

File: Events/Ch1.event [hunk 1]
```diff
@@ -102,46 +97,49 @@ UNIT Chiva Archer Vedge Level(1, Ally, 0) [10,1] 0x00 0x00 0x0 0x00 [0x00] NoAI
...
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [9,8] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [11,11] 0x00 0x00 0x0 0x00 [HandAxe] AttackInRangeAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [14,5] 0x00 0x00 0x0 0x00 [HandAxe] GuardTileAI
+UNIT Ch1Generic1 Mercenary Jazzak Level(2, Enemy, 1) [6,3] 0x00 0x00 0x0 0x00 [IronSword] AttackInRangeAI
+UNIT Ch1Generic1 Mercenary Jazzak Level(2, Enemy, 1) [12,7] 0x00 0x00 0x0 0x00 [IronSword] AttackInRangeAI
+UNIT Ch1Generic1 Shaman Jazzak Level(2, Enemy, 1) [14,6] 0x00 0x00 0x0 0x00 [Flux] GuardTileAI
+UNIT Ch1Generic1 Monk Jazzak Level(2, Enemy, 1) [1,5] 0x00 0x00 0x0 0x00 [Lightning] AttackInRangeAI
+UNIT Ch1Generic1 Mage Jazzak Level(2, Enemy, 1) [5,11] 0x00 0x00 0x0 0x00 [Fire] AttackInRangeAI
+UNIT Ch1Generic1 Shaman Jazzak Level(2, Enemy, 1) [16,3] 0x00 0x00 0x0 0x00 [Flux] GuardTileAI
+UNIT Jazzak Fighter 0x00 Level(4, Enemy, 0) [16,2] 0x00 0x00 0x0 0x00 [HandAxe] GuardTileAI
```
Comment (StanHash) [inline Events/Ch1.event]: `GuardTileAI` doesn't include the AI4 component needed for 0 movement display to work (so unless that's intentional, I suggest using `NeverMoveAI` instead)

File: Text/Chapter Text/Chapter 1.txt [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+[.v....]
+Vedge...[A][X]
+
+## Chapter1_NorthVillage
+[....v.][LoadCurrent]
+[.v....][LoadAwakeningAnna]
+Oh hey![N]
+You're the Nameless Knights of Void, right?[AN]
+My sisters have been talking about them.[N]
```
Comment (StanHash) [inline Text/Chapter Text/Chapter 1.txt]: Those lines enlarge the textbox quite a bit (to the point where the right edge touches the edge of the screen). The following lines end up only occupying about 2/3 of the space, making the whole thing look eh.

File: Events/Ch1.event [hunk 3]
```diff
@@ -102,46 +97,49 @@ UNIT Chiva Archer Vedge Level(1, Ally, 0) [10,1] 0x00 0x00 0x0 0x00 [0x00] NoAI
...
 UNIT
 
 Wave1:
-UNIT 0x80 Brigand Jazzak Level(4, Enemy, 1) [15,9] 0x00 0x00 0x0 0x00 [SteelAxe] BrigandAI
+UNIT Ch1Generic2 Brigand Jazzak Level(3, Enemy, 1) [15,9] 0x00 0x00 0x0 0x00 [IronAxe] BrigandAI
 UNIT
 
 Wave2:
-UNIT 0x80 Thief Jazzak Level(4, Enemy, 1) [17,9] 0x00 0x00 0x0 0x00 [SteelSword] LootingThiefAI
+UNIT Ch1Generic2 Thief Jazzak Level(3, Enemy, 1) [17,9] 0x00 0x00 0x0 0x00 [SteelSword] LootingThiefAI
```
Comment (StanHash) [inline Events/Ch1.event]: Looting thief without a way to loot (lockpick or chest key).

File: Text/Chapter Text/Chapter 1.txt [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+rates. All because he, like many[AN]
+other poor souls in this city,[N]
+descend from Donateenary immigrants[AN]
+that came here fleeing their own[N]
+country. Those people came to our[AN]
+nameless kingdom seeking freedom[N]
+and new opportunities. Instead, all[AN]
+they get is a life where violent[N]
+rebellion seems to be their only[AN]
+option...[A][2NL]
```
Comment (StanHash) [inline Text/Chapter Text/Chapter 1.txt]: ~~replace `[AN]`s with `[N]`s for the full Arch rant experience~~

File: Text/Chapter Text/Chapter 1.txt [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+my gratitude to your family.[AN]
+Even so, I've also never forgotten[N]
+my heritage as a commoner.[AN]
+Even after my sister Nada[N]
+accepted your marriage proposal,[AN]
+truly making us nobility,[N]
+I have not forgotten my heritage.[A]
+
+[....v.]
+So?[N]
```
Comment (StanHash) [inline Text/Chapter Text/Chapter 1.txt]: Is the absence of `[A]` intentional?

File: Text/Chapter Text/Chapter 1.txt [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+[.v....]
+So I can't help but feel sorry[N]
+for these people.[AN]
+I knew the "Donateenary Tax"[N]
+was a massive tax, but I had[AN]
+no clue it was this bad.[N]
+These people won't be able to[AN]
+survive much longer under[N]
+these conditions[A]
```
Comment (StanHash) [inline Text/Chapter Text/Chapter 1.txt]: punctuation?

File: Text/Chapter Text/Chapter 1.txt (hunk 4 above)
Comment (Crazycolorz5) [reply Text/Chapter Text/Chapter 1.txt]: I had that at first but it's actual good exposition so

**Pull request merged**